toolbox/commands/verify.py

```diff
@@ -53,3 +60,37 @@ def verify(context: typer.Context):
         rich.print(f"[red]{invalid_count} tasks are invalid.")
         raise Exit(code=1)
 
+
+def verify_assets(yaml_data: dict, assets_path: Path, subdir_path: Path) -> bool:
+    """
+    Verifies assets for a task.
+    """
+    assets = yaml_data.get("assets", [])
+
+    assets_paths = [asset["path"] for asset in assets]
+    if not find_unregistered_assets(assets_path, assets_paths, subdir_path):
+        return False
+    if not assets:
+        return True
+    if not assets_path.is_dir():
+        rich.print(f"[red]Missing assets directory for {subdir_path}")
+        return False
+
+    for asset in assets:
+        asset_path = assets_path / str(asset["path"])
+        if not asset_path.is_file():
+            rich.print(f"[red]Missing asset file {asset} for {subdir_path}")
+            return False
+
+    return True
+
+def find_unregistered_assets(assets_path: Path, assets_paths: list, subdir_path: Path) -> bool:
+    for asset in assets_path.iterdir():
+        if asset.is_file() and asset.name not in assets_paths:
```

Comment:
Umm, i think asset name is just name of file. So if we define in assets `skibidi/toilet.exe` it will just fail, because `toilet.exe` is not present. You can fix that or just remove sub directories entirely here, because I cannot think of scenario where they may be used.